enterprise/server/raft/filestore/filestore.go

```diff
@@ -68,6 +68,16 @@ func fileRecordSegments(r *rfpb.FileRecord) (partID string, groupID string, isol
 
 type PebbleKeyVersion int
 
+const (
+	// PebbleKeyVersion representing an unspecified key version. When this is
+	// the active key version, the database will select an active key version
+	// to use based on existing database contents and the maximum known key
+	// version.
+	//
+	// This is in a different const block so as not to affect the iota below.
+	UnspecifiedKeyVersion PebbleKeyVersion = -1
```

Comment:
To make this part of the pre-existing const block, you can do UnspecifiedKeyVersion PebbleKeyVersion = iota - 1. All the future consts will increment by 1

Reply:
That doesn't work because the const declaration increments the iota, changing the value of everything below it. I could add it at the bottom, and make it clear that new versions should be inserted between this const and the other versions, but thought this approach was simpler.

Reply:
Maggie is right, you can manipulate the value of iota by subtracting from it so you can certainly put it in the same const block.

Reply:
Ah, I misunderstood this and it wasn't working for me, but figured it out and updated. Sorry for the confusion!